.rubocop.yml

```diff
@@ -53,6 +53,20 @@ Style/ArgumentsForwarding:
   Enabled: true
 Style/CollectionCompact:
   Enabled: true
+Style/Documentation:
+  Exclude:
+    - '**/measure.rb'
+    - '**/tests/*.rb'
+    - 'HPXMLtoOpenStudio/resources/hpxml.rb'
+    - 'workflow/**/*.rb'
+    - 'tasks.rb'
+Style/DocumentationMethod:
+  Exclude:
+    - '**/measure.rb'
+    - '**/tests/*.rb'
+    - 'HPXMLtoOpenStudio/resources/hpxml.rb'
+    - 'workflow/**/*.rb'
+    - 'tasks.rb'
```

Comment:
Why are we excluding measure.rb files? I definitely would like the HPXMLtoOpenStudio/measure.rb file to get checked.